awswrangler/s3/_read_parquet.py

```diff
@@ -867,11 +873,37 @@ def read_parquet_table(
         path: str = location if location.endswith("/") else f"{location}/"
     except KeyError as ex:
         raise exceptions.InvalidTable(f"Missing s3 location for {database}.{table}.") from ex
+    path_root: Optional[str] = None
+    paths: Union[str, List[str]] = path
+    # If filter is available, fetch & filter out partitions
+    # Then list objects & process individual object keys under path_root
+    if partition_filter is not None:
+        available_partitions_dict = _get_partitions(
+            database=database,
+            table=table,
+            catalog_id=catalog_id,
+            boto3_session=boto3_session,
+        )
+        available_partitions = list(available_partitions_dict.keys())
+        if len(available_partitions) > 0:
+            paths = []
+            path_root = path
+            partitions: Union[str, List[str]] = _apply_partition_filter(
+                path_root=path, paths=available_partitions, filter_func=partition_filter
```

Comment:
Should it not be `path_root=path_root`? Realize it's equivalent but just for consistency

Reply:
Yeah, that'll do, too.